src/Schema/Builder.php

```diff
@@ -35,19 +35,13 @@ class Builder extends BaseBuilder
 
 
     /**
-     * @return list<string>
+     * @return list<array{ name: string, type: string }>
      */
     public function getAllTables()
     {
-        /** @var list<array{ TABLE_NAME: string }> $results */
-        $results = $this->connection->select(
+        return $this->connection->select(
             $this->grammar->compileGetAllTables()
         );
-
-        /** @var Processor $processor */
-        $processor = $this->connection->getPostProcessor();
-
-        return $processor->processGetAllTables($results);
```

Comment:
https://github.com/colopl/laravel-spanner/blob/0a9ca9811486dfd45e5eabb2a5c72edd37c2e704/src/Query/Processor.php

Would you also remove `Processor::processGetAllTables`?
(It's a breaking change anyway...)

Reply:
fixed